server.js

```
@@ -0,0 +1,116 @@
+"use strict";
+
+const http = require("http");
+const fs = require("fs");
+const elemTemplate = require("./elemTemplate.js");
+const indexTemplate = require("./indexTemplate.js");
+const querystring = require("querystring");
+const PORT = 8080;
+const server = http.createServer((req, res) => {
+  req.setEncoding("utf8");
+
+  // START OF GET
+  if (req.method === "GET") {
+    let URL = req.url;
+    if (URL === "/") {
+      URL = "/index.html";
+    }
+    let getType = URL.split(".");
+    let contentType = getType[1]; // grab "html" or "css"
+    // check the public directory
+    fs.readdir("./public/", (err, dir) => {
+      if (err) {
+        return console.log(err);
```

This should return an error to the client using either a 400/500 error